src/Cli/Microsoft.TemplateEngine.Cli/Commands/InteractiveMode/InteractiveMode.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.CommandLine;
+using System.CommandLine.Invocation;
+
+namespace Microsoft.TemplateEngine.Cli.Commands.InteractiveMode
+{
+    public static class InteractiveMode
+    {
+        public static async Task EnterInteractiveMode(InvocationContext context, Func<InvocationContext, Task> next)
+        {
+            if (context.ParseResult.CommandResult.Command.Name == "new")
```

Comment:
this is identical to `context.ParseResult.CommandResult.Command is NewCommand` so next statement will never be true.